docs/devguide/newdashboards.asciidoc

```diff
@@ -297,6 +210,61 @@ If Elasticsearch is running on a different host, then you can use the `ES_URL` v
 ES_URL="http://192.168.3.206:9200" make export-dashboards
 ----------------------------
 
+/////////////////////
+
+==== Exporting Kibana 6.0 dashboards and newer
+
+The `dev-tools/cmd/export_dashboards.go` script helps you export your customized Kibana 6.0 dashboards and newer. You might need to export a single dashboard 
+
+
+===== Export a single Kibana dashboard
+
+You can export a single dashboard by passing the dashboard ID in the `-dashboard` flag.
+
+NOTE:: The dashboard ID is available in the dashboard URL. For example, in case the dashboard URL is
```

Comment:
To get the correct note formatting, use `NOTE:` (a single colon). Otherwise, you won't get the correct formatting and icon. (There are multiple instances of NOTE:: that need to be fixed.)

Reply:
done